access/lims.py

```diff
@@ -1,76 +1 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-#Script that connects to the MySQL database and parses data from an html table
-#Import the mysql.connector library/module
-import sys
-import time
-import glob
-import re
-import os
-import requests
-from xml.etree import ElementTree
 
-def readconfig():
-  configfile = os.getenv('HOME') + '/.scilifelabrc'
-  params = {}
-  with open(configfile, "r") as confs:
-    for line in confs:
-      if len(line) > 3 and not line[0] == "#":
-        pv = line.rstrip().split(" ")
-        arg = pv[0]
-        pv.pop(0)
-        params[arg] = ' '.join(pv)
-  return params
-
-class limsconnect(object):
-  
-  def __init__(self, User, Passwd, baseuri):
-    self.user = User
-    self.pwd = Passwd
-    self.uri = baseuri
-    self.entrypoints = { 'samples': 'limsid', 'artifacts': 'limsid' }
-
-  def __enter__(self):
-    return self
-
-  def __exit__(self, exc_type, exc_val, exc_tb):
-    if exc_type:
-      print '__exit__(%s, %s, %s)' % (exc_type, exc_val, exc_tb)
```

Comment:
I've been transitioning over to use the format minilanguage which is the new Python standard. We should consider using it throughout our Python code or not, hehe!